documentation/components/AutoMapper.rst

```diff
@@ -164,6 +164,7 @@ Then configure the bundle to your needs, for example:
 
     jane_auto_mapper:
       autoregister: true
+      date_time_format: !php/const \DateTimeInterface::RFC3339_EXTENDED
```

Comment:
Please add a quick description bellow in the "Possible fields" section ?

Reply:
Done! :ok_hand: